docker/bin/prepare.bash

```diff
@@ -63,6 +63,7 @@ LOG_DIR="$ARMADILLO_ROOT/log/"
 mkdir -p "$LOG_DIR" || exit 1
 
 cp "$PROJECT_DIR/scripts/release/release-test.R" "$BIN_DIR/" || exit 1
+cp -r "$PROJECT_DIR/scripts/release/release-test-cases" "$BIN_DIR/" || exit 1
```

Comment:
Nice dir name ... maybe make it shorter
- from`/release/release-test-cases`
- into `/release/test-cases`

That makes less names with `r` in it so easier to type expand in the terminal